code/modules/reagents/reagent_containers/borghydro.dm

```diff
@@ -70,8 +70,10 @@
 			return
 
 	if (M.can_inject(user, 1))
-		user << "<span class='notice'>You inject [M] with the injector.</span>"
-		M << "<span class='notice'>You feel a tiny prick!</span>"
+		visible_message("<span class='notice'>[user] starts to inject [M] with their hypospray!</span>")

```

Comment:
`mob/visible_message` works differently from other visible_message protos. It has a 2nd argument which is the same as going `user << ""`. So this should be changed to:
`visible_message("<span class='notice'>[user] starts to inject [M] with their hypospray!</span>", "<span class='notice'>You start to inject [M] with the injector.</span>")`